core/pkg/monitor/cpu.go

```diff
@@ -62,7 +72,12 @@ func (c *CPU) SampleMetrics() {
 
 	// total system CPU usage in percent
 	utilization, err := cpu.Percent(0, true)
-	if err == nil {
+	if err != nil {
+		// do not log "not implemented yet" errors
+		if !strings.Contains(err.Error(), "not implemented yet") {
```

Comment:
Maybe define an error and use the Is function?

Reply:
I'm checking what gopsutils returns... and it just returns a errors.errorString

Reply:
That’s annoying